Source/control.cpp

```diff
@@ -176,6 +176,61 @@ const char *const PanBtnStr[8] = {
 	"" // Player attack
 };
 
+const char *const SpellDescription[MAX_SPELLS] = {
+	N_("null"),
+	N_("Creates a magical flaming missile"),
+	N_("Instantly restores lost life"),
+	N_("Creates a powerful lightning bolt to lay waste to your enemies"),
+	N_("Releases a short-ranged but deadly burst of energy"),
+	N_("Identify a magic item's properties"),
+	N_("Creates a wall of flame that burns nearby enemies"),
+	N_("Create a magic portal to town"),

```

Comment:
```suggestion
	N_("Creates a magic portal to town"),
```
You went with "creates" everywhere else

Reply:
Perhaps we should actually remove the `s` throughout? It's cleaner

Reply:
> Perhaps we should actually remove the `s` throughout? It's cleaner
It sounds more like an instruction than a description without the plural case